package.json

```diff
@@ -14,8 +14,9 @@
   },
   "scripts": {
     "build": "turbo run build",
```

Comment:
would build benefit from clean as well? I recall running into this while testing for deployments

Reply:
Yes I think so, I never ran into that issue but it makes sense to start each build in a clean state, just updated to add clean to build